hangouts_chat.py

```diff
@@ -391,7 +392,16 @@ def send_message(self, message):
             if thread_id:
                 message_payload['thread'] = {'name': thread_id}
 
-            self.chat_api.create_message(space_id, message_payload, thread_key)
+            gc = self.chat_api.create_message(space_id, message_payload, thread_key)
+
+            # errbot expects no return https://errbot.readthedocs.io/en/latest/errbot.core.html#errbot.core.ErrBot.send_message
+            # but we need this in order to get the thread_id from a thread_key generated message
+
+            return None if gc == None else {
+                'space_id': gc['space']['name'],
```

Comment:
in case we start getting something back other than `None` that doesn't match the expected data shape, I think it's safer to switch these to `gc.get('space', {}).get('name', '')` to avoid lookup KeyErrors.